### ISSUE TITLE
How to get the probablity in each multiclass?

### ISSUE THREAD

**Issue by jerronl** [Yura52/rtdl#49]
other than the max bucket from argmax, can we also have the probabilities for each class? currently the prediction often have a lot negative values and I don't know what would be the right way to convert them to probabilities.

**Comment by Yura52** [comment]
The predictions are **logits**. To convert them to **probabilities**, use [softmax](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html):

```
import torch.nn.functional as F

logits = model(x)
probabilities = F.softmax(logits, dim=-1)
```
